source/renderer/app/components/wallet/summary/WalletSummary.js

```diff
@@ -31,7 +32,8 @@ const messages = defineMessages({
 
 type Props = {
   wallet: Wallet,
-  numberOfTransactions: number,
+  recentTransactionsLength: number,
```

Comment:
@daniloprates please remove this prop into `numberOfRecentTransactions`

Reply:
✅